umbra-js/test/Umbra.test.ts

```diff
@@ -54,6 +58,11 @@ describe('Umbra class', () => {
     sender.connect(ethers.provider);
     receiver = ethers.Wallet.fromMnemonic(mnemonic as string, `${path as string}/${receiverIndex}`);
     receiver.connect(ethers.provider);
+    receivers.push(receiver);
+    for (let i = 4; i < 10; i++) {
+      receivers.push(ethers.Wallet.fromMnemonic(mnemonic as string, `${path as string}/${i}`));
+      receivers[i - 3].connect(ethers.provider);
```

Comment:
Actually you can just start the loop at 0 to make this indexing simpler, then in the derivation path just use `i + 100`, which is probably the most readable IMO

Reply:
Made the changes here: https://github.com/ScopeLift/umbra-protocol/pull/426/commits/ddc768e7504ccb601877993210dbfa445f644277